5_People/career.md

```diff
@@ -102,13 +102,21 @@ Everyone tracks working hours for themselves. At the end of each month, add your
 
 ## Severance Policy
 
+### Termination
+
 While many of us are good friends, we are still a business, not a family. Unlike a family, people join and leave us from time to time. Whenever we have to let someone go, this is how we do it.
 
-* The person needs to know at least 1 month in advance that their position could be canceled in the near term.
+* The person is given three notices, the third one being the final chance. There should be minimum one month in between notices, approved by one founder. A notice expires after six months.
```

Comment:
```suggestion
* The person is given three notices, the third one being the last chance. There should be a minimum of one month between notices, approved by one founder. A notice expires after six months.
```

Reply:
If a notice expires after six months, and three are required before termination, then there isn't any room there, is it? Three notices spanning 2 months, already the first one expires?

Reply:
Maybe 

> Notices expire a year after the last one was given.